### ISSUE TITLE
Pictures shows differently in Gutenberg vs Classical on smartphone

### ISSUE THREAD

**Issue by AstaC** [WordPress/gutenberg#9207]
**Describe the bug**
When I attach an Image with Gutenberg and use the same size as when I use the classic texteditor the Image gets a smaller size when you look at the image on your smartphone.
See attached images.

**To Reproduce**
With Gutenberg
Create image block, chose Large (would be very good if that could be a set choice like it is today) and insert image.

With Classical texteditor.
Insert Image - Large.

**Expected behavior**
Image should have the same size when you look at them in the same browser.  I don't want them smaller
**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**
 - Device: [Note 8]
 - OS: [Android 8.1]
 - Browser [Chrome]
 - Version [nesest]

**Additional context**
- Please add the version of Gutenberg you are using in the description. 3.6.2
Theme - Genesis 2.6.1

![classical_20180821-194552_chrome](https://user-images.githubusercontent.com/42585130/44420109-5c219580-a57d-11e8-8f58-4c7320d85d0e.jpg)
![gutenberg_20180821-194505_chrome](https://user-images.githubusercontent.com/42585130/44420110-5c219580-a57d-11e8-8fdd-977477d5fcae.jpg)

- To report a security issue, please visit the WordPress HackerOne program: https://hackerone.com/wordpress.

**Comment by designsimply** [comment]
Noting this may be a duplicate of #6177 and needs investigation.

May I also ask for you to note your theme and theme version for reference in case it becomes relevant?

**Comment by AstaC** [comment]
Theme Genesis   -    version 2.6.1 . (childtheme  Pretty Creative)

**Comment by designsimply** [comment]
I tested and (at first) I found that I didn't have any problems and couldn't see any difference in large images inserted with the classic editor vs Gutenberg. Then I re-tested and found that "Image Size" in the Gutenberg image block settings doesn't respect the Settings > Media > Image sizes options. May I ask if you have modified those settings in the past? If you have, we can file this issue as a request to update that. If you are still using the default image sizes in media settings, then further troubleshooting may still be needed.

![screen shot 2018-08-21 at tue aug 21 5 40 20 pm](https://user-images.githubusercontent.com/1119271/44434613-5a4dd700-a569-11e8-94fc-5453e3923310.png)
<sup>Seen at http://alittletestblog.com/wp-admin/options-media.php running WordPress 4.9.8 and Gutenberg 3.6.2 using Firefox 61.0.2 on macOS 10.13.6.</sup>

**Comment by AstaC** [comment]
Yes, I have changed the media settings in the past. And I noticed that Gutenberg only suggested thumbnail as format event though i used Large a several times. When I chose large the images is said to get the right size (in this case 800x600) - see code from my test.
```
<!-- wp:image {"id":24999} -->
<figure class="wp-block-image"><img class="wp-image-24999" src="https://wp-content/uploads/2018/08/20171231_120515-01.jpeg?fit=800%2C600" alt="" /></figure>
<!-- /wp:image -->
```

It looks in the code as though the images have the same size ( in this case 800x600) and it even seems so when I look in the editor (see first  image) but when I look on the screen it says 740x540 (see second image).

![test](https://user-images.githubusercontent.com/42585130/44454570-5ebad400-a5eb-11e8-9354-b411be586db4.JPG)

![test 1](https://user-images.githubusercontent.com/42585130/44454569-5e223d80-a5eb-11e8-9319-d12bdb103ce3.JPG)
